components/automate-gateway/gateway/services.go

```diff
@@ -475,14 +476,15 @@ func (s *Server) ProfileCreateHandler(w http.ResponseWriter, r *http.Request) {
 		file, _, err := r.FormFile("file")
 		if err != nil {
 			log.Errorf("Received error while getting file from request : %s", err.Error())
-			http.Error(w, err.Error(), http.StatusBadRequest)
+			http.Error(w, "Error getting file", http.StatusBadRequest)
 			return
 		}
 		defer file.Close() // nolint: errcheck
 
 		_, err = io.Copy(&content, file)
 		if err != nil {
-			http.Error(w, err.Error(), http.StatusBadRequest)
+			log.Errorf("Received error while copying content : %s", err.Error())
+			http.Error(w, "Something went wrong", http.StatusBadRequest)
```

Comment:
please add appropriate message instead of some thing went wrong

Reply:
@Dmaddu This is requested by security team.

Reply:
Yes, @Dmaddu these error msg are as per discussion with security team, 
<img width="583" alt="Screenshot 2023-08-10 at 10 58 02 AM" src="https://github.com/chef/automate/assets/38317160/c598d7dc-1ccd-43aa-aea3-528ab019f348">